docs/development.md

```diff
@@ -78,4 +78,76 @@ define the WATCH_NAMESPACE environment variable.
 
 ```
 WATCH_NAMESPACE=3scale make run
-```
\ No newline at end of file
+```
+
+## Publishing a new release
```

Comment:
I see documentation about how to do a new release.

Operator image is created like always with circle ci, after creating a release tag on GitHub (who executes make target `build-push-operator-image`. I guess that circleci should be updated to use new make target `build-and-publish` (updating circleci to not execute that on master merges like now...).

But who is in charge of generating the CSV with the correct operator image, correct `version` field...., maybe the new bundle stuff + kustomize is in charge of that? What about the CSV with the `replace` field of previous version (so it does upgrades, this is very important)?

Reply:
As discussed over slack, `version` and `replaces` are managed by the makefile targets. We will eventually add a snapshot of the `/bundle` in the catalog repo.